alf/optimizers/optimizers_test.py

```diff
@@ -252,6 +254,64 @@ def _test(i, sampled_predictive=False):
         self.assertLess(mean_err, 0.5)
         self.assertLess(cov_err, 0.5)
 
+    @parameterized.parameters(
+        dict(capacity_ratio=0.5, masked_out_value=None, opt_steps=3),
+        dict(capacity_ratio=0.5, masked_out_value=0, opt_steps=5))
+    def test_capacity_scheduling(self, capacity_ratio, masked_out_value,
+                                 opt_steps):
+        layer = torch.nn.Linear(5, 3)
```

Comment:
use a large parameter and check the mask is approximately close to the desired capacity.

Reply:
Added